src/colp/components/agency_usetype_report.py

```diff
@@ -0,0 +1,78 @@
+import streamlit as st
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import plotly.express as px
+
+
+class CountRecordsReport(ABC):
+    def __call__(self):
+        if self.data is None:
+            st.info(f"No Count of records by {self.X_axis_col}")
+            return
+        by_agency = ""
+        middle_str = ""
+        by_usetype = ""
+        if self.by_agency:
+            by_agency = " agency"
+        if self.by_usetype:
+            by_usetype = " use type"
+        if self.by_agency and self.by_usetype:
+            middle_str = " and"
+        st.subheader(
+            f"City owned and leased properties by {by_agency}{middle_str}{by_usetype}",
+            anchor="corrections-applied",
+        )
+        self.plot()
+
+    def plot(self):
+        x_lim = st.number_input(
```

Comment:
I hit the same issue about there being a ton of things, but i feel like we need to do something else to segment it. I wonder if this could work?
https://plotly.com/python/range-slider/

Reply:
Alternatively, if we wanted to keep this number selection thing, I would maybe use 
https://docs.streamlit.io/library/api-reference/widgets/st.slider

it feels weird that you cant choose to look at the lower end of the distribution as it is

Reply:
It also might end up being easier to show a larger set of values to make this horizontal instead (my strategy was to make it horizontal and very tall so you could scroll through it, but that has its own pros and cons)